src/cluster.c

```diff
@@ -212,13 +213,17 @@ int clusterLoadConfig(char *filename) {
             n = createClusterNode(argv[0],0);
             clusterAddNode(n);
         }
+
+        /*Human readable name*/
+        memcpy(n->human_readable_name, argv[1], strlen(argv[1]) + 1);
```

Comment:
If a human readable name is not set, this looks like it will break. We still need to be backwards compatible. I don't really have answer to this since the cluster nodes format doesn't allow additional fields since slots is variadic. We could add more optional fields at the end.